numpy/lib/npyio.py

```diff
@@ -1465,7 +1464,7 @@ def first_write(self, v):
 
 @set_module('numpy')
 def fromregex(file, regexp, dtype, encoding=None):
-    """
+    r"""
```

Comment:
Don't see the need for this, the escaped items render fine in iPython and the documentation without it.

Reply:
Ah, you're right, this change wasn't strictly necessary.  I only ran into a problem because I was copy-pasting directly from within the docstring in my editor to an interactive python session, so I ended up with the double backslashes.